Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/threshold/ThresholdedRandomCutForest.java

```diff
@@ -290,24 +560,27 @@ protected boolean trigger(DiVector candidate, int gap, int baseDimension, DiVect
      * @return description containing scores, grade, confidence, expected values,
      *         attribution etc.
      */
-    protected AnomalyDescriptor getAnomalyDescription(double[] point) {
+    protected AnomalyDescriptor getAnomalyDescription(double[] point, long inputTimeStamp, double[] inputPoint) {
         AnomalyDescriptor result = new AnomalyDescriptor();
         DiVector attribution = forest.getAnomalyAttribution(point);
         double score = attribution.getHighLowSum();
         result.setRcfScore(score);
```

Comment:
You could modify the definition of `setAttribution` in AnomalyDescriptor to update the rcfScore value as well.

Reply:
True -- since we have two pieces of information we can diverge. But It's possible someone does not want attribution ... even though they have the rest of the fields ... say 1d input where only the time stamp is changing (mixture of gaussians)